### ISSUE TITLE
Encrypt wallet failure

### ISSUE THREAD

**Issue by Sosthene00** [bisq-network/bisq#877]
Upgraded from Bitsquare on 16th of july, tried to encrypt the new Bisq wallet by creating a password, but the app didn't react properly, I got an error message that unfortunately I didn't record. By the time I was in a hurry, just closed the app and decided to check that later.

Today, upgraded from 0.5.1 to 0.5.3. Didn't ask me any password while starting, beside the "wallet password" interface looked like the wallet wasn't encrypted, so I just assumed it failed the last time and went for creating a password, and then it said the wallet was already encrypted. Problem is, even if I type the password I used the first time, nothing works. I tried to withdraw the bitcoins from my wallet, and got this error message : 

Error at creating transaction: org.bitcoinj.core.ECKey$KeyIsEncryptedException

[bisq.txt](https://github.com/bitsquare/bitsquare/files/1167258/bisq.txt)

**Comment by ManfredKarrer** [comment]
Did you try to import the seed words from the old wallet (version < 0.5.0)? If so that does not work as the wallet format has changed and is not backward compatible. 
If you have no funds, trades, offers in the new app, simple delete the app directory and start over again. You need to transfer the funds to the new app by a normal btc tx.

**Comment by Sosthene00** [comment]
Yes I tried to import the old wallet, only to find out later that it's not backward compatible. But it's not really the issue here, sorry I didn't make myself very clear. 

The real problem is that it messed up something while trying to encrypt the wallet, and I end up having an encrypted wallet that seemed not to be acknowledged as such in the app. I'm sorry I can't be more precise than that, I really don't have a clue. 

Anyway I succeeded import the wallet on another computer and took the remaining funds away, so I guess it will be alright. Maybe we can close the issue, except if someone else thinks there's some issue here and want to investigate.
